data/json/items/magazines.json

```diff
@@ -54,6 +54,40 @@
         ]
     },
     {   "type" : "MAGAZINE",
+        "id" : "lp_45_mag",
+        "price" : 5800,
+        "name" : "L39-45 magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A compact light-weight magazine for use with L39-45 and L39B-45 pistols.",
+        "material": ["steel", "plastic"],
+        "volume" : 1,
+        "weight" : 70,
+        "to_hit" : -2,
+        "ammo_type" : "45",
+        "capacity" : 12,
+        "reliability" : 10,
+        "reload_time" : 100,
+        "flags": [ "COMPACT_MAG" ]
```

Comment:
I must have forgotten to remove those mod slots! You aren't supposed to be able to convert them; they come in two different calibers for a reason.